.github/workflows/link-checker-pr.yml

```diff
@@ -43,7 +43,7 @@ jobs:
       run: |
         sed -i '1,/---------------------------------------/d' linkreport.txt
         echo 'LINKREPORT<<EOF' >> $GITHUB_ENV
-        cat linkreport.txt >> $GITHUB_ENV
+        head -c 65300 linkreport.txt >> $GITHUB_ENV
```

Comment:
Nice! Not used the `head` command before, just out of curiosity, how come we set **65300** specifically here?

Reply:
Apparently the source of the failure is that the maximum length of a GitHub comment (and/or the `$GITHUB_ENV` variable??) is 65536 (that's 2<sup>16</sup>) characters long. This somewhat arbitrary cutoff of 65300 gives enough headroom for the little bit of extra text the job adds to the comment aside from the link report itself. I based it on [this similar issue](https://github.com/robburger/terraform-pr-commenter/issues/6#issuecomment-826966670).